script.js

```diff
@@ -1,28 +1,32 @@
 $(document).ready(function(){
 
-
-
-
   var lightSeq = [];
   var turn = 0;
-  var attempts = 0;
-
+  var attempt = 1;
```

Comment:
All of the state that is being maintained in this function (`.ready`) makes me think that we're missing an abstraction here to help manage the state...